README.md

```diff
@@ -114,10 +114,12 @@ Before filing an issue, please be sure to read the guidelines for what you're re
 
 ### I'm using JSCS, should I migrate to ESLint?
 
-Maybe, depending on how much you need it. [JSCS has reached end of life](https://eslint.org/blog/2016/07/jscs-end-of-life), but if it is working for you then there is no reason to move yet. There are still [a few issues](https://github.com/eslint/eslint/milestones/JSCS%20Compatibility) pending. We’ll announce when all of the changes necessary to support JSCS users in ESLint are complete and will start encouraging JSCS users to switch to ESLint at that time.
+Yes, if feasible. [JSCS has reached end of life](https://eslint.org/blog/2016/07/jscs-end-of-life), but if it is working for you then there is no reason to move yet. That said, we believe we are near 100% compatibility with JSCS at this point.
```

Comment:
I don't think this change conveys the right message. It's not just feasible to migrate to ESLint, we recommend it, because otherwise you are using unsupported software.

Reply:
@ilyavolodin Good feedback, thanks. Please see the latest commit.